hosted/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV DEPLOYMENT_STAGE=$HAPPY_ENV
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y gettext nginx moreutils build-essential libxml2-dev python3-dev python3-pip zlib1g-dev python3-requests python3-aiohttp llvm jq npm git zip curl && \
+    apt-get install -y gettext nginx moreutils build-essential libxml2-dev python3-dev python3-pip zlib1g-dev python3-requests python3-aiohttp llvm jq git zip curl && \
```

Comment:
I think `npm` was wrong here. This installed a node version that was then overridden by the following install line. @tihuan can you verify if this makes sense?

Reply:
ooooh good call! Yeah that sounds right to me. And if the app compiles then LGTM lol